demisto_sdk/commands/update_release_notes/update_rn.py

```diff
@@ -429,6 +429,28 @@ def get_master_version(self) -> str:
             master_current_version = master_metadata.get("currentVersion", "0.0.0")
         return master_current_version
 
+    def get_pack_display_name(self) -> str:
+        """
+        Gets the pack display name from the metadata file. If not found return the pack name.
+
+        :rtype: ``str``
+        :return
+            Pack display name
+
+        """
+        pack_display_name = self.pack
+        master_metadata = None
+        try:
+            master_metadata = get_remote_file(self.metadata_path)
+        except Exception:
+            logger.exception(
+                f"[red]Failed fetching {self.metadata_path} from remote master branch."
+                "Using the local version (if exists), instead[/red]",
+            )
+        if master_metadata:
+            pack_display_name = master_metadata.get("name")
+        return pack_display_name
```

Comment:
We should have the pack name inside the pack model (`pack.name`), no need to re-implement this especially not in the update.rn

Reply:
Removed this function.
In this PR we still not using the Pack object, but we have already a function that gets the metadata.